com.github.utsushi.Utsushi.metainfo.xml

```diff
@@ -25,9 +25,7 @@
   <description xml:lang="pt_BR"><p>Preserve suas fotos e documentos importantes criando cópias digitais nunca foi tão fácil! Tenha seu scanner conectado ao seu computador , abra este programa e comece à digitalizar. Quando uma foto ou documento é digitalizado, é super fácil compartilhá-la por e-mail, enviá-la para sua rede social favorita ou colocá-la em um pen-drive. Este programa certamente te ajudará a realizar digitalizações com precisão, clareza e no formato de arquivo que você precisa! **(Este aplicativo só funciona com scanners da Epson).**</p></description>
   <description xml:lang="es"><p>¡Ahora preservar tus fotos y documentos importantes es más fácil que nunca! Solo conecta el escáner a tu computadora, abre este programa y comienza a escanear. Una vez que un documento o una foto este digitalizado, compartirlos es igual de fácil. Puedes enviarlos por correo electrónico, cargarlos en tu red social favorita o ponerlos en una memoria USB. Este programa te permitirá asegurarte de que todo lo que escanees sea claro, recto y en el formato que necesitas. **(Esta aplicación sólo funciona con escáneres de Epson).**</p></description>
   <screenshots>
-    <screenshot>
-      <image type="source">https://raw.githubusercontent.com/flathub/com.github.utsushi.Utsushi/master/screenshots/1.png</image>
-    </screenshot>
+      <screenshot><image width="786" height="616">https://raw.githubusercontent.com/flathub/com.github.utsushi.Utsushi/master/screenshots/1.png</image></screenshot>
```

Comment:
not sure why you change this.

Reply:
I thought, image width and height are necessary of Flathub? Is it not?

I couldn't see the image, but it is probably the Flathub bug I linked above.